CMakeLists.txt

```diff
@@ -47,7 +47,9 @@ set_property(CACHE OTTO_BOARD PROPERTY STRINGS ${OTTO_BOARDS})
 if (OTTO_USE_LIBCXX)
   message("Using libc++ instead of libstdc++")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -stdlib=libc++")
-  set(CMAKE_LINKER_FLAGS "${CMAKE_LINKER_FLAGS} -lc++fs")
+  set(CMAKE_LINKER_FLAGS "${CMAKE_LINKER_FLAGS} -lc++fs"
+    $<$<AND:$<NOT:$<PLATFORM_ID:Windows>>,$<AND:$<CXX_COMPILER_ID:Clang>,$<AND:$<VERSION_GREATER_EQUAL:$<CXX_COMPILER_VERSION>,7>,$<VERSION_LESS:$<CXX_COMPILER_VERSION>,9>>>>:-lc++fs>
+    )
```

Comment:
```suggestion
```
We require clang 10 at this point anyway, so just remove this all together

Reply:
Sounds good, I've removed this all.